Create Quiz schema

abdulkadiret opened the pull request.

abdulkadiret committed:
Install morgan

abdulkadiret committed:
Create QuizSchema & Quiz model

abdulkadiret committed:
Merge branch origin master

abdulkadiret committed:
Formate to single quotes using prettier

bonboh commented:
@abdulkadiret Does this PR add any new code now that I've merged in PR https://github.com/CodeYourFuture/project-quiz/pull/72/?

abdulkadiret committed:
Fix merge conflict

abdulkadiret commented:
I resolved the merge conflict. Initially I had written the codes on one branch, then I extracted out the codes into different branches according to the issues and to help the reviewing. Unfortunately, I forgot to extract them all out from PR#72, I thought i did.  
There are small changes in this branch tho like:
- I renamed the file models/`quizzes.js` -----> into -----> models/`quiz.js`
- I renamed the schema name from `Quiz` into `QuizSchema` and the model name from `CyfQuiz` into `Quiz`

bonboh commented:
Okay no problem, I'll go ahead and merge this 👍
